src/Core/Content/Product/SalesChannel/Listing/CachedProductListingRoute.php

```diff
@@ -1,116 +0,0 @@
-<?php declare(strict_types=1);
-
-namespace Shopware\Core\Content\Product\SalesChannel\Listing;
-
-use Shopware\Core\Content\Product\Events\ProductListingRouteCacheKeyEvent;
-use Shopware\Core\Content\Product\Events\ProductListingRouteCacheTagsEvent;
-use Shopware\Core\Framework\Adapter\Cache\AbstractCacheTracer;
-use Shopware\Core\Framework\Adapter\Cache\CacheValueCompressor;
-use Shopware\Core\Framework\DataAbstractionLayer\Cache\EntityCacheKeyGenerator;
-use Shopware\Core\Framework\DataAbstractionLayer\Field\Flag\RuleAreas;
-use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
-use Shopware\Core\Framework\Feature;
-use Shopware\Core\Framework\Log\Package;
-use Shopware\Core\Framework\Util\Hasher;
-use Shopware\Core\System\SalesChannel\SalesChannelContext;
-use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\Routing\Attribute\Route;
-use Symfony\Contracts\Cache\CacheInterface;
-use Symfony\Contracts\Cache\ItemInterface;
-use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
-
-#[Route(defaults: ['_routeScope' => ['store-api']])]
-#[Package('inventory')]
-class CachedProductListingRoute extends AbstractProductListingRoute
```

Comment:
I guess this one was simply missed. after doing the change of the feature flag, this class does nothing. 
I would suggest to do a deprecation for 6.6